### PR TITLE
Supporting Kai on Workspace Creation

### PR DESCRIPTION
**Pull request by noprysk-ua**
I took https://github.com/singlestore-labs/terraform-provider-singlestoredb/pull/29 and made the CI green

### CODE REVIEW

File: internal/provider/workspaces/resource.go [hunk 1]
```diff
@@ -288,6 +298,13 @@ func (r *workspaceResource) ModifyPlan(ctx context.Context, req resource.ModifyP
 		return
 	}
 
+	if !plan.KaiEnabled.Equal(state.KaiEnabled) {
+		resp.Diagnostics.AddError("Cannot change the kai_enabled configuration for the workspace",
+			"Changing the kai_enabled configuration is currently not supported.")
+
+		return
+	}
+
```
Comment (JMonteiro13) [inline internal/provider/workspaces/resource.go:307]: does this mean that you can't change kai settings after you create the workspace?
if so, why is that?

File: internal/provider/workspaces/updatesizeorsuspended.go [hunk 2]
```diff
@@ -57,7 +57,7 @@ func scale(ctx context.Context, c management.ClientWithResponsesInterface, plan
 
 func resume(ctx context.Context, c management.ClientWithResponsesInterface, plan workspaceResourceModel) (workspaceResourceModel, *util.SummaryWithDetailError) {
 	id := uuid.MustParse(plan.ID.ValueString())
-	workspaceResumeResponse, err := c.PostV1WorkspacesWorkspaceIDResumeWithResponse(ctx, id)
+	workspaceResumeResponse, err := c.PostV1WorkspacesWorkspaceIDResumeWithResponse(ctx, id, management.WorkspaceResume{})
```
Comment (JMonteiro13) [inline internal/provider/workspaces/updatesizeorsuspended.go:60]: is this change intended? Seems to affect resume but the PR is for Create changes

File: internal/provider/workspaces/resource.go (hunk 1 above)
Comment (noprysk-ua) [reply internal/provider/workspaces/resource.go:307]: The Kai team decided not to support it

File: internal/provider/workspaces/updatesizeorsuspended.go (hunk 2 above)
Comment (noprysk-ua) [reply internal/provider/workspaces/updatesizeorsuspended.go:60]: Yes, because we bumped the version of the SDK and it needs it now

**Review by JMonteiro13 — APPROVED** [review]
